src/globals.js

```diff
@@ -86,4 +102,5 @@ export const clearGlobals = () => {
   SpansContainer.clearSpans();
   TracerGlobals.clearTracerInputs();
   TracerGlobals.clearHandlerInputs();
+  GlobalTimer.clearTimer();
```

Comment:
why export is needed here?

Reply:
Which export? for clearGlobals?
most of the exports are for testing